frontend/src/components/Login/Login.tsx

```diff
@@ -55,8 +55,8 @@ export function Login() {
           <InstagramIcon />
         </div>
         <form className="login-form flex-direction-column" onSubmit={handleSubmit}>
-          <Input type="email" placeholder="Phone number, username, or email" name="email" onChange={onChange} />
-          <Input type="password" placeholder="Password" name="password" onChange={onChange} />
+          <InputField type="email" placeholder="Phone number, username, or email" name="email" onChange={onChange} />
```

Comment:
I think the placeholder should be "Email", as we are only logging in via email and the type of the input element is also email.